source/pele/cell_list_potential.h

```diff
@@ -316,7 +316,47 @@ class EnergyGradientHessianAccumulator {
     }
 };
 
+/**
+ * class which accumulates the pressure tensor one pair interaction at a time
+ */
+/*
+template <typename pairwise_interaction, typename distance_policy>
+class PressureTensorAccumulator {
```

Comment:
should you move this to a different file, more specific to pressure calculations?